packages/askui-nodejs/src/lib/ui-controller-linux.ts

```diff
@@ -1,10 +1,35 @@
 import { exec } from 'child_process';
+import { promisify } from 'util';
 import { UiControllerFacade } from './ui-controller-facade';
 import { logger } from './logger';
+import { WaylandError } from './wayland-error';
+import { LibfuseError } from './libfuse-error';
 
 export class UiControllerLinux extends UiControllerFacade {
   // eslint-disable-next-line class-methods-use-this
   protected override makeBinaryExecutable(): void {
     exec(`chmod +x "${this.binaryPath}"`, (_exception, stdout) => logger.debug(stdout));
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  protected override async preStartChecks(): Promise<void> {
+    const runCommand = promisify(exec);
+    const waylandStatus = await runCommand('echo $WAYLAND_DISPLAY');
+
+    if (waylandStatus.stdout.trim().includes('wayland')) {
+      throw new WaylandError('Wayland is not supported: https://docs.askui.com/docs/general/Troubleshooting/askui-ui-controller-starting-problems#wayland');
+    }
+
+    try {
+      await runCommand('dpkg --version');
+    } catch (err) {
+      return;
+    }
```

Comment:
can you please explain to me what is this doing?

Reply:
this is checking if we dpkg is used (if user has debian distribution). First i only checked if ubuntu is used. But Maxi and Dominik mentioned this problem can also occur on mint etc.

Reply:
sorry, my question was regarding the return;

Reply:
if it is not using dpkg/ no debian distribution. I want to end the check and don't want the libfuse2 control anymore
because `dpkg -s libfuse2 | grep Status` would always fail

Reply:
then i assume you can remove this?

Reply:
I think I need this? method return if there is no dpkg and should not check for libfuse2

Reply:
so the final result we want to check for libfuse2. 
if `dpkg --version` fails, we can't check for libfuse2, and you assume that it will be there, and that's why you are using the return ?